doc/nrf/app_dev/bootloaders_dfu/mcuboot_nsib/bootloader_adding_sysbuild.rst

```diff
@@ -263,7 +263,8 @@ You can set the option in :file:`sysbuild.conf` or using the command line:
 
       Escaped quotations avoid malformed-string warnings from Kconfig.
 
-The path of the key must be an absolute path, though ``${APPLICATION_CONFIG_DIR}`` can be used to get the path of the application configuration directory to use keys relative to this directory.
+The path of the key must be an absolute path.
+The To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``).
```

Comment:
```suggestion
To utilize keys relative to the application configuration directory, use the escaped CMake variable ``\${APPLICATION_CONFIG_DIR}`` in the path by replacing ``$`` with ``\$``.
```

Reply:
Silly mistake that, fixed

Reply:
the leftover bracket wasn't removed - can we get rid of that?

Reply:
Now should be fixed